wolfcrypt/src/rsa.c

```diff
@@ -1143,21 +1153,21 @@ static int RsaPad_OAEP(const byte* input, word32 inputLen, byte* pkcsBlock,
         return ret;
     }
 
+#ifdef WOLFSSL_SMALL_STACK
```

Comment:
Check `pkcsBlockLen - hLen - 1` <= `RSA_MAX_SIZE/8 + RSA_PSS_PAD_SZ`.

Reply:
Already checked in RsaPublicEncryptEx()